public/usage-examples/geometry/random_bitmap_point/random_bitmap_point-1-simple-oop.cs

```diff
@@ -0,0 +1,30 @@
+using SplashKitSDK;
+
+namespace RandomBitmapPoint
+{
+    public class Program
+    {
+        public static void Main()
+        {   
+            //Create Window
+            SplashKit.OpenWindow("random bitmap point", 600, 600);
+            Bitmap bmp = new Bitmap("random quads",600,600);
+
+            // create quad using random points on bitmap
+            Quad q = SplashKit.QuadFrom(
+                SplashKit.RandomBitmapPoint(bmp),
+                SplashKit.RandomBitmapPoint(bmp),SplashKit.RandomBitmapPoint(bmp),

```

Comment:
make the second one in here be on a new line, this part - `SplashKit.RandomBitmapPoint(bmp)`